VideoLocker/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -1184,7 +1185,18 @@ public RegisterResponse register(Bundle parameters)
         }
         logger.debug("Register response= " + json);
 
+        //the server side response format is not client friendly ... so..
```

Comment:
This message could be clearer. Maybe something like "Checks against a 'successful' response with an error message in the json"